recipes/libsndfile/1.0.29/conanfile.py

```diff
@@ -0,0 +1,115 @@
+    homepage = "http://www.mega-nerd.com/libsndfile"
+    description = "Libsndfile is a library of C routines for reading and writing files containing sampled audio data."
+    topics = ("audio")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "programs": [True, False],
+        "experimental": [True, False],
+        "with_alsa": [True, False],
+        "with_sqlite": [True, False],
+        "with_external_libs": [True, False]
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "programs": True,
+        "experimental": False,
+
+        "with_alsa": False,
+        "with_sqlite": True,
+        "with_external_libs": True,
+    }
+    exports_sources = "CMakeLists.txt"
+    generators = "cmake", "cmake_find_package"
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def requirements(self):
+        if self.options.get_safe("with_alsa"):
+            self.requires("libalsa/1.2.2")
+        if self.options.with_sqlite:
+            self.requires("sqlite3/3.32.3")
+        if self.options.with_external_libs:
+            self.requires("flac/1.3.3")
+            self.requires("ogg/1.3.4")
+            self.requires("opus/1.3.1")
+            self.requires("vorbis/1.3.7")
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+            del self.options.with_alsa
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        os.rename("{}-{}".format(self.name, self.version), self._source_subfolder)
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+        self._cmake.definitions["BUILD_PROGRAMS"] = self.options.programs
+        self._cmake.definitions["BUILD_EXAMPLES"] = False
+        self._cmake.definitions["BUILD_TESTING"] = False
+        self._cmake.definitions["ENABLE_CPACK"] = False
+        self._cmake.definitions["ENABLE_EXPERIMENTAL"] = self.options.experimental
+        if self.settings.compiler == "Visual Studio":
+            self._cmake.definitions["ENABLE_STATIC_RUNTIME"] = "MT" in str(self.settings.compiler.runtime)
+        self._cmake.definitions["BUILD_REGTEST"] = False
+        self._cmake.definitions["ENABLE_EXTERNAL_LIBS"] = self.options.with_external_libs
+        self._cmake.configure()
+        return self._cmake
+
+    def build(self):
+        cmake = self._configure_cmake()
+        cmake.build()
+
+    def package(self):
+        self.copy("COPYING", dst="licenses", src=self._source_subfolder)
+        cmake = self._configure_cmake()
+        cmake.install()
+
+        tools.rmdir(os.path.join(self.package_folder, "lib", "cmake"))
```

Comment:
```suggestion
        tools.rmdir(os.path.join(self.package_folder, "lib", "cmake"))
        tools.rmdir(os.path.join(self.package_folder, "cmake"))
```

Reply:
@MartinDelille 
I think the hooks are complaining about the `cmake` directory on Windows.